content/en/docs/Concepts/constraints.md

```diff
@@ -6,31 +6,31 @@ description: >
   How Luet turns Image resolution into CSP
 ---
 
-Luet uses SAT under the hood and Reinforcement learning techniques to solve package constraints.
+Under the hood, Luet uses boolean satisfiability problem ([SAT](https://en.wikipedia.org/wiki/Boolean_satisfiability_problem))  [reinforcement learning](https://en.wikipedia.org/wiki/Reinforcement_learning) techniques to solve package constraints.
 
-Luet allows you to specify 3 kind of set of contraints on a [package](/docs/docs/concepts/packages/) definition:
+Luet allows you to specify 3 types of set of contraints on a [package](/docs/docs/concepts/packages/) definition:
 
 - Requires
 - Conflicts
 - Provides
 
-The package definition in your tree definition along with its Requires and Conflicts are turned into Boolean formulas that are consumed by the solver to compute a solution. The solution represent the state of your system after a particular query is asked to the solver (Install, Uninstall, Upgrade).
+The package definition in your tree definition, along with its Requires and Conflicts, are turned into Boolean formulas that are consumed by the solver to compute a solution. The solution represent the state of your system after a particular query is asked to the solver (Install, Uninstall, Upgrade).
 
 ## Requires and Conflicts
 
-A list of requires and conflicts, composed of one or more [packages](/docs/docs/concepts/packages/), becomes a SAT formula. The formula is then given to the SAT solver to compute a finite state set of packages that must be installed in the system to met the requirements.
+A list of requires and conflicts, composed of one or more [packages](/docs/docs/concepts/packages/), becomes a SAT formula. The formula is then given to the SAT solver to compute a finite state set of packages which must be installed in the system in order to met the requirements.
 
-Along, Luet builds the ALO (At least one) and AMO (At most one) constraints. That means that given requirements as selectors in semver notation (e.g. `>1.0`) it builds additional formulas to restrict the constraints of the given solution.
+Alongside, Luet builds the ALO (At least one) and AMO (At most one). <!-- I think this sentence should be rewritten-->This means that given requirements as selectors in semver notation (e.g. `>1.0`) it builds additional formulas to restrict the constraints of the given solution.
```

Comment:
:+1: I'm not sure if comments in html are parsed, but will rewrite it on top of this PR